src/provider/camunda-platform/properties/ConditionProps.js

```diff
@@ -333,6 +339,93 @@ function Resource(props) {
   />;
 }
 
+function VariableEventProps(props) {
+  const entries = [];
+
+  entries.push({
+    id: 'conditionVariableName',
+    component: VariableName,
+    isEdited: isTextFieldEntryEdited
+  });
+
+  entries.push({
```

Comment:
IMO not for StartEvents (compare existing implementation)

Reply:
Should also add a testcase for that

Reply:
Good catch with the start event. I made it conditional

https://github.com/bpmn-io/bpmn-js-properties-panel/blob/b623f6d8c79ada663c62ace4939d658dbafc0a4e/src/provider/camunda-platform/properties/ConditionProps.js#L355-L361


And added the ConditionalStartEvent to the test case 
https://github.com/bpmn-io/bpmn-js-properties-panel/blob/b623f6d8c79ada663c62ace4939d658dbafc0a4e/test/spec/provider/camunda-platform/ConditionProps.spec.js#L537-L540